### PR TITLE
Added "Back to Top" Button

### LINKED ISSUE #1194: Add "Back to Top" Button to the README.md
I would like to add a button in the README.md file for "Back to Top". It would be highly beneficial for contributors to navigate through the README.md file more easily, ultimately increasing the time spent reading the repository's README.md file.

Example:

BacktoTop

## Please have a look and assign this issue to me under SSOC 2.0

### PR DESCRIPTION
**Pull request by kunalnandre**
Added "Back to Top" Button to the README.md

## 🛠️ Issue (#1194)

Closes #1194

# Description

I have added a button in the README.md file for "Back to Top". It would be highly beneficial for contributors to navigate through the README.md file more easily, ultimately increasing the time spent reading the repository's README.md file.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have made this from my own
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

Backtt

### CODE REVIEW

**Review by harshit-paneri — COMMENTED** [review]
Give Project deployment link / live link.

**Comment by kunalnandre** [comment]
I have added a button in the README.md file for "Back to Top".

**Review by harshit-paneri — APPROVED** [review]
Very small change!!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Front-End-Projects
+<div id="top">
```
Comment (TusharKesarwani) [inline README.md:2]: Close this div element

File: README.md [hunk 2]
```diff
@@ -28,6 +29,8 @@
 ![GitHub pull requests](https://img.shields.io/github/issues-pr/TusharKesarwani/Front-End-Projects)
 ![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/TusharKesarwani/Front-End-Projects)
  </p>
+ </p>
```
Comment (TusharKesarwani) [inline README.md:32]: Remove this closing tag

**Comment by TusharKesarwani** [comment]
Make fresh PR
With proper changes

**Pull request closed without merging**